apps/playground/src/app/components/ag-grid/readonly-grid/readonly-grid.component.ts

```diff
@@ -140,21 +139,18 @@ export class ReadonlyGridComponent implements OnInit {
     this.gridApi = gridReadyEvent.api;
     this.gridApi.sizeColumnsToFit();
     this.gridApi.resetRowHeights();
-    this.gridApi.addEventListener(
-      Events.EVENT_ROW_SELECTED,
```

Comment:
`Events.EVENT_*` are replaced via the schematic. AG Grid switched to a string union type.